e2e/test/prerequisites/iot_config.cmd_template

```diff
@@ -38,6 +38,22 @@ set IOTHUB_EVENTHUB_COMPATIBLE_NAME=
 @REM EventHub consumer group:
 set IOTHUB_EVENTHUB_CONSUMER_GROUP=
 
+@REM ---------- ASCforIoT Oms Configuration ----------
```

Comment:
Please provide instructions here: https://github.com/Azure/azure-iot-sdk-csharp/tree/master/e2e/test/prerequisites, on how to set these values.